File: src/main/java/ru/arty_bikini/crm/dto/orders/PersonalDataDTO.java
package ru.arty_bikini.crm.dto.orders;

import ru.arty_bikini.crm.data.dict.TrainerEntity;
import ru.arty_bikini.crm.data.enums.ClientLanguage;
import ru.arty_bikini.crm.dto.dict.TrainerDTO;

import javax.persistence.*;
import java.time.LocalDateTime;

//класс данных секретных
public class PersonalDataDTO {
    private String family;//фамилия
    private String name;//имя
    private String patronymic;//отчество

    private String prepayment;//предоплата
    private String payment;//цена
    private String telephon;//телефон
    private String city;//город
    private ClientLanguage language;
    private String addOrder;//допольнительно к заказу

    private LocalDateTime OrderTime;//дата заказа(предоплата или заполнение мерок?)
    private LocalDateTime CompetitionTime;//дата соревнований
    private LocalDateTime neededTime;//дата, когда нужен заказ

    private String comment;//комментарий на чем остановились

    private TrainerDTO trainer;//тренер

    public TrainerDTO getTrainer() {
        return trainer;
    }

    public String getFamily() {
        return family;
    }

    public String getName() {
        return name;
    }

    public String getPatronymic() {
        return patronymic;
    }

    public String getPrepayment() {
        return prepayment;
    }

    public String getPayment() {
        return payment;
    }

    public String getTelephon() {
        return telephon;
    }

    public String getCity() {
        return city;
    }

    public ClientLanguage getLanguage() {
        return language;
    }

    public String getAddOrder() {
        return addOrder;
    }

    public LocalDateTime getOrderTime() {
        return OrderTime;
    }

    public LocalDateTime getCompetitionTime() {
        return CompetitionTime;
    }

    public LocalDateTime getNeededTime() {
        return neededTime;
    }

    public String getComment() {
        return comment;
    }

    public void setFamily(String family) {
        this.family = family;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setPatronymic(String patronymic) {
        this.patronymic = patronymic;
    }

    public void setPrepayment(String prepayment) {
        this.prepayment = prepayment;
    }

    public void setPayment(String payment) {
        this.payment = payment;
    }

    public void setTelephon(String telephon) {
        this.telephon = telephon;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public void setLanguage(ClientLanguage language) {
        this.language = language;
    }

    public void setAddOrder(String addOrder) {
        this.addOrder = addOrder;
    }

    public void setOrderTime(LocalDateTime orderTime) {
        OrderTime = orderTime;
    }

    public void setCompetitionTime(LocalDateTime competitionTime) {
        CompetitionTime = competitionTime;
    }

    public void setNeededTime(LocalDateTime neededTime) {
        this.neededTime = neededTime;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public void setTrainer(TrainerDTO trainer) {
        this.trainer = trainer;
    }
}

File: common/src/commonMain/kotlin/ru/arty_bikini/crm/dto/orders/PersonalDataDTO.kt
package ru.arty_bikini.crm.dto.orders

import kotlinx.serialization.Serializable
import ru.arty_bikini.crm.dto.dict.TrainerDTO
import ru.arty_bikini.crm.dto.enums.personalData.ClientLanguage

//класс данных секретных
@Serializable
class PersonalDataDTO {
    var name: String? = null //фамилия имя отчество
    var telephon: String? = null //телефон
    var city: String? = null  //город
    var language: ClientLanguage? = null
    
    var prepayment: String? = null  //предоплата
    var payment: Int = 0  //цена
    var addOrder: String? = null //дополнительно к заказу
    var comment: String? = null //комментарий на чем остановились
    
    var packageNow = false
    
    var createdTime : Long? = null //дата появления лида
    var archiveTime : Long? = null //дата попадания в архив
    var orderTime: Long? = null //дата заказа(предоплата или заполнение мерок?)
    var packageTime : Long? = null //Дата, когда нужно отправить
    
    var packageManager : Long? = null ///дата, когда нужно отправить, которую указал менеджер(менеджер дата отправки)
    var packageManufacture : Long? = null //дата, которую посчитала система (4 недели пошив от даты заполнения мерок)
    var packageClient : Long? = null //дата, которую посчитала система, (клиент указал дату нужности или соревнований + время на пошив)
    
    var deliveryTime : Int = 0 //время доставки
    var userPackage: String? = null
    
    var trainer: TrainerDTO? = null  //тренер
    var payTrainer = false
    var payMoney =0
    var sale = 0//скидка
    
    
}